lighttpd/datadog_checks/lighttpd/lighttpd.py

```diff
@@ -89,7 +89,12 @@ def check(self, instance):
             self.service_check(self.SERVICE_CHECK_NAME, AgentCheck.OK, tags=service_check_tags)
 
         headers_resp = r.headers
-        server_version = self._get_server_version(headers_resp)
+        full_version, server_version = self._get_server_version(headers_resp)
+        if full_version:
```

Comment:
Probably better to be explicit here:

```suggestion
        if full_version is not None:
```

(I've seen several places where we check for `is None`/`is not None` with a simple boolean check, but it's not always entirely clear whether we're also encapsulating checks for false-y values, such as `0`, `[]`, etc. So I guess I'm on my way to evangelize explicit `None` checks now? 😂)

Reply:
Haha fixed!